packages/familie-backend/src/auth/authenticate.ts

```diff
@@ -64,6 +64,25 @@ export const ensureAuthenticated = (
     };
 };
 
+export const ensureOBOAccessTokenValid=(
+    saksbehandlerTokenConfig: ITokenRequest,
+    oboTokenConfig?: ITokenRequest,
+)=>{
+    return async(req: Request, res: Response, next: NextFunction) =>{
```

Comment:
I dont think we need to save the token to the session. Just get the token and use it when saving it to the session.

Reply:
ok